docs/admin/open-in-coder.md

```diff
@@ -0,0 +1,29 @@
+# Open in Coder Button
+
+This is a Markdown button you can add to your project's `README.md` to get your developers up and running with Coder twice as fast.
+
+A basic example looks like this:
+
+```text
+[![open in coder button](https://cdn.coder.com/embed-button.svg)](https://dev.coder.com/?template=coder-ts)
+```
+
+which renders like this:
+
+![open in coder button](https://cdn.coder.com/embed-button.svg)
+
+You can customize this to take developers directly to your team's template. Read on to learn more.
+
+### Customization
+
+The underlying link for this button consists of the following pieces:
+- <deployment-url>: where your Coder deployment lives i.e. https://dev.coder.com
+- <query-params>: optional query parameters to customize the experience
```

Comment:
Why not add each query parameter here?

Reply:
As in replace `<query-params>` with each query param here? I think I know what you're saying but feel free to add a suggestion!